BrainPortal/app/helpers/select_box_helper.rb

```diff
@@ -118,6 +118,22 @@ def group_select(parameter_name = "group_id", options = {}, select_tag_options =
       selected = selector.to_s
     end
 
+    # for Admin filter out only public, invisible or non_assignable to reduce the clutter
+    if current_user.has_role? :admin_user
+      current_user_id = current_user.id
```

Comment:
What is this line for?

Reply:
removed. I guess was flipping between using all the admin ids vs the present admin id